Makefile

```diff
@@ -213,7 +211,7 @@ github-release:
 
 
 build-docker: ## Build docker image
-	./ci_scripts/docker-push.sh -t ${BRANCH} -b
+	./ci_scripts/docker-push.sh -t latest -b
```

Comment:
This needs to be `prod`

Reply:
I dont think so! that is in general docker build process! which should be tagged latest.